db/migrate/20230505010811_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :author_id
+      t.string :title
+      t.string :text
```

Comment:
- [ ] For the post content, a `string` column would be insufficent. `:string` is mapped to `VARCHAR` and has a limit of 255 characters. 
As a general rule of thumb, use `:string` for short text input (username, email, password, titles, etc.) and use `:text` for longer expected input such as descriptions, post body, comment content etc.